swagger.yaml

```diff
@@ -639,4 +748,47 @@ components:
           description: Selected week of on call entries
         paid:
           type: boolean
-          description: Status of on call week payment
\ No newline at end of file
+          description: Status of on call week payment
+
+    SoftwareStatus:
+      type: string
+      enum: [pending, under review, accepted, deprecated, declined]
+      description: Status of the software entry
```

Comment:
Here is an example of how an enum schema has been defined in the time-bank, I would recommend doing this in a similar way:

    Timespan:
      type: string
      description: Enum for timespan
      enum:
        - ALL_TIME
        - YEAR
        - MONTH
        - WEEK